core/src/main/java/com/linecorp/armeria/common/DefaultRequestHeadersBuilder.java

```diff
@@ -112,4 +118,35 @@ public RequestHeadersBuilder authority(String authority) {
         setters().authority(authority);
         return this;
     }
+
+    @Override
+    public List<LanguageRange> acceptLanguages() {
+        final HttpHeadersBase getters = getters();
+        if (getters != null) {
+            return getters.acceptLanguages();
+        }
+        return DefaultRequestHeaders.ACCEPT_LANGUAGE_WILDCARD;
+    }
+
+    public RequestHeadersBuilder acceptLanguages(List<LanguageRange> acceptLanguages) {
+        requireNonNull(acceptLanguages, "acceptLanguages");
+        final String acceptLanguagesValue = acceptLanguages
+                .stream()
+                .map(LanguageRange::toString)
+                .collect(Collectors.joining(", "));
+        set(
+                HttpHeaderNames.ACCEPT_LANGUAGE,
+                acceptLanguagesValue
+        );
```

Comment:
Let's merge these into one line because this is Java. 🤣  
`set(HttpHeaderNames.ACCEPT_LANGUAGE, acceptLanguagesValue);`